### PR TITLE
Blog: v7.0.0 release post

### PR DESCRIPTION
**Pull request by jasnell**
Refs: https://github.com/nodejs/node/pull/9099

### CODE REVIEW

**Comment by MylesBorins** [comment]
LGTM

**Comment by jasnell** [comment]
landed!

File: locale/en/blog/release/v7.0.0.md [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+
+Node.js v7 has arrived!
+
+This new major version of Node.js includes: V8 5.4 which brings along with 98% coverage of ES6 language features, improved reliability and performance, and a new experimental URL parser based on the [WHATWG URL standard](https://url.spec.whatwg.org/).
+
+As an odd numbered release, in accordance with our [Long Term Support plan](https://github.com/nodejs/lts), Node.js v7 will be supported only until about June, 2017, with Node.js v8 currently scheduled for release in April, 2017.
+
+The general rule for deciding which version of Node.js to use is:
+
+* Stay on Node.js v4 or upgrade to Node.js v6 if you need *stability and have a complex production environment, e.g. you are a medium or large enterprise.
```
Comment (addaleax) [inline locale/en/blog/release/v7.0.0.md:19]: @jasnell Stray `*` before `stability`?

Comment (stevemao) [reply locale/en/blog/release/v7.0.0.md:19]: https://github.com/nodejs/nodejs.org/pull/994

**Pull request closed without merging**